app/CocoaPods/CPUserProject.m

```diff
@@ -266,6 +271,13 @@ - (void)appendTaskOutput:(NSString *)rawOutput;
   } else {
     self.taskOutput = attributedOutput;
   }
+
+  // Keep the text view at the bottom if it was previously, otherwise restore the previous position.
+  if (scrolledToBottom) {
+    [self.progressOutputView scrollToEndOfDocument:self];
```

Comment:
Perfect (not too familiar with this OS X API), just checking because otherwise this logic wouldn't work when lines are printed while the underlying view is animated.

:+1: